tools/ci_build/build.py

```diff
@@ -493,13 +493,15 @@ def run_onnx_tests(build_dir, configs, onnx_test_data_dir, provider, enable_para
           cmd.append(model_dir)
         if os.path.exists(onnx_test_data_dir):
           cmd.append(onnx_test_data_dir)
-        run_subprocess([exe] + cmd, cwd=cwd)
```

Comment:
why do you remove this?

Reply:
I moved it to line 503. Looks like we were running the same test twice when parallel test is requested. Was there any reason for that?

Reply:
The arguments are different. 
If enable_parallel_executor_test is true, we need to run the tests twice:
1. with parallel executor enabled ('-x')
2. without parallel executor ( without '-x')

And, your changes  disabled multiple threaded test(run multiple inferences at the same time), this is the biggest concern I have.

Reply:
sent an update.